codegen/gateway.go

```diff
@@ -41,6 +41,14 @@ func getDirName() string {
 	return zanzibar.GetDirnameFromRuntimeCaller(file)
 }
 
+var lintCorrectionMap = map[string]string{
```

Comment:
is it possible to have import some golint rules here? I'm a bit concern about us maintaining a key word list, but :man_shrugging: